test/e2e/config/docker.yaml

```diff
@@ -92,9 +92,9 @@ variables:
   KUBERNETES_VERSION_UPGRADE_TO: "v1.19.1"
   KUBERNETES_VERSION_UPGRADE_FROM: "v1.18.2"
   DOCKER_SERVICE_DOMAIN: "cluster.local"
-  DOCKER_SERVICE_CIDRS: "10.128.0.0/12"
+  DOCKER_SERVICE_CIDRS: "fd00:100:64::/108"
```

Comment:
This file is being used by all the tests, so I think we should avoid to change the CIDR here; instead we can use one of the two alternatives:
- overriding the default value using env variables provided before running the test (all the test will run in ipv6 mode)
- overriding the default value using env variables set inside a specific test (only a specific test will run in ipv6 mode)

Reply:
resolving because I have summarised consideration about ipv6 E2E testing down in a comment